CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-## [_Unreleased_](https://github.com/freckle/freckle-app/compare/v1.2.0.0...main)
+## [_Unreleased_](https://github.com/freckle/freckle-app/compare/v1.2.0.1...main)
+
+## [v1.2.0.1](https://github.com/freckle/freckle-app/compare/v1.2.0.0...v1.2.0.1)
+
+- Use `Env.kept` with parsing that occurs in `makePostgresPool`
+
+  This ensure all `PG` variables are kept in the environment after parsing, to
+  again match pre-v1.1 behavior.
+
+  If you would prefer not to keep these variables, parse them yourself (e.g.
+  with `envParseDatabaseConf`) and use `makePostgresPoolWith`.
```

Comment:
Would this be more useful in the haddock?

Personally I look there before changelogs for usage info like this.

Reply:
I'm not sure. To me, this behavior is only notable _because it changed_ (hence CHANGELOG). The behavior as it was/is isn't particularly surprising to me, not worth calling out in docs at least. :shrug: